src/App.js

```diff
@@ -1,9 +1,13 @@
 // import '../style file';
```

Comment:
Great job @enmanuelbayen 🤝

- [ ] Kindy remove commented code from your project as this is regarded as a bad practice.